docs/ossec_alerts.rst

```diff
@@ -527,18 +551,43 @@ securedrop@freedom.press for help.
 
 .. _SecureDrop Support Portal: https://securedrop-support.readthedocs.io/en/latest/
 
-Uncommon OSSEC Alerts
+Daily reports
+^^^^^^^^^^^^^
+
+On days where file integrity checksums have changed or users have logged into ``app``
+or ``mon`` servers, you will receive emails entitled ``Daily report: File changes`` or
+``Daily report: Successful logins``. These emails may be a more convenient format
+should you not have continuous access to the inbox or GPG key.
+
+**Action**: periodically review these daily reports to ensure file changes correspond
+to platform updates and logins correspond to authorized admin activity on the SecureDrop
+servers.
+
+If you have any suggestions on how to further tune or improve the alerting,
+you can open an issue on `GitHub <https://github.com/freedomofpress/securedrop/labels/goals%3A%20reduce%20IDS%20noise>`__.
+
+Uncommon OSSEC alerts
 ~~~~~~~~~~~~~~~~~~~~~
 
-.. _uncommon_alerts:
+Data integrity
+^^^^^^^^^^^^^^
 
-SecureDrop also runs automatic checks for submission data integrity
+SecureDrop runs automatic checks for submission data integrity
 problems. For example, secure deletion of large submissions could
-potentially be interrupted, resulting in an alert recommending steps
-to :ref:`clean them up <submission-cleanup>`.
+potentially be interrupted ::
```

Comment:
This is a terrible nit, but the space before the colons means no colon will be rendered:

![ossec-docs-missing-colon](https://user-images.githubusercontent.com/47430598/102554528-04cf5980-4093-11eb-85d4-e869b11edf26.png)

Reply:
good catch, fixed